specification/schemaregistry/SchemaRegistry/client.tsp

```diff
@@ -0,0 +1,62 @@
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/versioning";
+import "./main.tsp";
+
+using TypeSpec.Versioning;
+using Azure.ClientGenerator.Core;
+using SchemaRegistry;
+
+@TypeSpec.Versioning.useDependency(
+  ServiceApiVersions.V2023_07_01
+)
+namespace ClientCustomizations;
+
+@client({name: "SchemaRegistryClient", service: SchemaRegistry})
+interface SchemaOperations {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard read operation w/ SchemaId resource doesn't allow returning binary schema and headers."
+  listSchemaGroups is SchemaRegistry.listSchemaGroups;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard read operation w/ Version resource doesn't allow returning non-version response."
+  listSchemaVersions is SchemaRegistry.listSchemaVersions;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard read operation w/ SchemaId resource doesn't allow returning binary schema and headers."
+  getSchemaById is SchemaRegistry.getSchemaById;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard read operation w/ Version resource doesn't allow returning non-version response."
+  getSchemaByVersion is SchemaRegistry.getSchemaByVersion;
+  #suppress "@azure-tools/typespec-azure-core/byos" "storage service is not used"
+  getSchemaPropertiesByContent is SchemaRegistry.getSchemaPropertiesByContent;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard createOrReplace operation does not allow passing in binary body to request."
+  #suppress "@azure-tools/typespec-azure-core/byos" "storage service is not used"
+  registerSchema is SchemaRegistry.registerSchema;
+}
+
+@@access(SchemaGroup, Access.internal);
+
+// csharp modifications
+@doc("The content type for the schema.")
+enum ContentTypeEnum {
```

Comment:
There was a recent update that changes the way enums are defined for Azure services. We need to get this updated before merging this PR in, docs: https://azure.github.io/typespec-azure/docs/libraries/azure-core/rules/no-enum#-correct